contents/docs/libraries/wordpress.md

```diff
@@ -4,60 +4,96 @@ icon: >-
   https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/integrate/frameworks/wordpress.svg
 ---
 
-Getting analytics about your [WordPress](https://www.wordpress.org/) site is simple with PostHog. Get data about traffic, usage, and user behavior with our free, open-source analytics platform. Once you have that data, you can discover insights and build dashboards with our suite of analytics tools.
+Getting traffic, usage, and user behavior data about your [WordPress](https://www.wordpress.org/) site is simple with PostHog. Once you have that data, you can discover insights and build dashboards with our suite of analytics tools.
 
-In this guide, we'll walk you through how to set up PostHog on your WordPress site. You can do this by adding our JavaScript snippet to the header of your WordPress site directly, or by using a third-party plugin to insert the snippet into your site's header.
+## How to add PostHog to your WordPress site
 
-## TL;DR: (the short version)
+The best way to add PostHog to your WordPress site depends on what version of WordPress you are using. 
 
-1. [Signup for PostHog Cloud](https://app.posthog.com/signup).
+1. After [signing up for PostHog](https://us.posthog.com/signup), get your [snippet](/docs/getting-started/install?tab=snippet) with your project API key and instance address from [your project settings](https://us.posthog.com/settings/environment-details#snippet).
 
-2. Get your [PostHog snippet](/docs/integrate?tab=snippet) from your 'Project Settings' or the initial PostHog setup.
 
-3. Add the PostHog snippet before the closing `</head>` tag in your `header.php` template file. This can be done in two ways:
-    1. Access WordPress admin, navigate to 'Appearance' -> 'Theme Editor', select your theme, and select 'Theme Header'.
-    2. Manually update the header template file. This is usually `wp-content/themes/your-theme/header.php`.
+All of them require you to add the PostHog snippet with your project API key and instance address. You can get this from your project settings.
 
-Read on for additional information.
+### Option 1: Add a theme function
```

Comment:
This had me scratching my head as a long-time WP hacker until I read the following graf. Would you take

### Option 1: Edit your theme's functions file

or nah?